Sources/SweetTableController.swift

```diff
@@ -24,6 +26,16 @@
             self.addConstraints()
         }
 
+        open override func viewWillAppear(_ animated: Bool) {
+            super.viewWillAppear(animated)
+
+            if self.clearsSelectionOnViewWillAppear {
```

Comment:
`clearsSelectionOnViewWillAppear` is available for UITableViewController subclasses, we don't have that. And it also defaults to false, we can make it default to true, though.